app/views/converts/new.slim

```diff
@@ -0,0 +1,18 @@
+- title 'convert'
+- content_for(:header) do
+  h2 convert splits to other cool formats
+article
+  form#convert method="post" action="" enctype="multipart/form-data"
+    .form-group.col-sm-4
+      input type="file" name="file"
+    .form-group.col-sm-2
+      select name="format"
+        - (Run.programs - [Llanfair]).each do |prgrm|
+          option value="#{prgrm.to_sym}"
+            = prgrm.to_s
+        option value="json" JSON
+    .form-group.col-sm-2
+      input type="checkbox" name="historic" id="historic"
+      label for="historic" Include History?
+    .form-group.col-sm-2
+      input type="submit" value="convert" id="convert-submit"
```

Comment:
Add `class="btn btn-default"` to this to make it look like the button on `/games`.